src/js/components/Anchor/__tests__/__snapshots__/Anchor-test.js.snap

```diff
@@ -4,7 +4,6 @@ exports[`Anchor disabled renders 1`] = `
 .c0 {
   font-size: 18px;
   line-height: 24px;
-  color: #444444;
```

Comment:
why are all these color #444444 going away?

Reply:
We only set the color at the top level if we set the background color in the theme now.